.github/actions/fetch_ctk/action.yml

```diff
@@ -112,16 +110,16 @@ runs:
         populate_cuda_path cuda_cudart
         populate_cuda_path cuda_nvrtc
         populate_cuda_path cuda_profiler_api
-        populate_cuda_path libnvjitlink
+        populate_cuda_path cuda_cccl
```

Comment:
This is needed by #261. Turns out that the cooperative group headers depend on `nv/target`, the latter of which is part of CCCL but the former is not... 🙁 (cc @jrhemstad)